### ISSUE TITLE
Can't send email checking

### ISSUE THREAD

**Issue by pokapow** [FreshRSS/FreshRSS#2927]
When I try to create new account with email checking, I get this error in logs:

`[error] --- Minz_Mailer cannot send a message: Could not instantiate mail function.`

In GUI when I clic in resend email button I got:

`We were unable to send you an email due to a misconfiguration of the server.`
(translate from french...)

I'm using official docker-compose file of FreshRSS version 1.16.1-dev

**Comment by Alkarex** [comment]
Indeed, when adding e-mail, we have not documented how to set it up.

There are multiple ways to do it, cf. e.g. those discussions:
* https://github.com/docker-library/php/issues/135
* https://github.com/docker-library/wordpress/issues/30

What do you say @marienfressinaud ?

**Comment by marienfressinaud** [comment]
It is documented, but it may be incomplete: https://github.com/FreshRSS/FreshRSS/blob/master/docs/en/admins/05_Configuring_email_validation.md

**Comment by Alkarex** [comment]
Ah, sorry @marienfressinaud , I could not remember we had that :-)
@pokapow I hope that helps?

**Comment by math-GH** [comment]
Any news here? How to proceed here?
